xcode/config_generator/render_xcconfigs.rb

```diff
@@ -0,0 +1,98 @@
+require 'json'
+require 'mustache'
+
+
+# create config files if needed
+system("touch configs_data.json") unless File.exist?("configs_data")
```

Comment:
shell scripting in ruby :/
create sh script or use ruby File API

Reply:
👍